metrics/metrics.go

```diff
@@ -172,13 +175,17 @@ func CollectProcessMetrics(refresh time.Duration) {
 		secondsSinceLastCollect := collectTime.Sub(lastCollectTime).Seconds()
 		lastCollectTime = collectTime
 		if secondsSinceLastCollect > 0 {
-			sysLoad := (cpustats[now].GlobalTime - cpustats[prev].GlobalTime) / secondsSinceLastCollect
-			sysWait := (cpustats[now].GlobalWait - cpustats[prev].GlobalWait) / secondsSinceLastCollect
-			procLoad := (cpustats[now].LocalTime - cpustats[prev].LocalTime) / secondsSinceLastCollect
+			sysLoad := (cpustats[now].GlobalTime - cpustats[prev].GlobalTime) * 100
+			sysWait := (cpustats[now].GlobalWait - cpustats[prev].GlobalWait) * 100
+			procLoad := (cpustats[now].LocalTime - cpustats[prev].LocalTime) * 100
```

Comment:
It's a bit weird to multiply by 100 here. The `* 100` is the conversion to percent. Multiplying here will make the counter values have a unit of centiseconds, which is not very common. Best to record the counter without multiplication so the unit will be seconds.

Reply:
I originally wanted to go with seconds, but  counters as implemented in geth only support integer values.  I just need to wrap my mind around  whether that is going to significantly reduce the accuracy of the counter.

Reply:
Hehe, I had the same instinct at first, but then figured it's actually fine. The total CPU time used by geth will most likely be a very large number, even when its in seconds. Sub-second precision is not important here, I think.

If you really to care about it, I'd be open to making it milliseconds (`* 1000`).

Reply:
Or we could add a float counter.

Reply:
ms is also fine. But then we should rename the metrics to `totalms` or something.

Reply:
Actually, I don't think seconds will work.  While it's true the total accumulated value will be large and fractions won't matter, when you call the `Inc()` function, truncating to integer really bites you. i mean, anything less than 1 get truncated to zero.  I suppose rounding would help a bit, but still not ideal.

So probably ms, or seconds with a float counter.  The first option is way less work.

Reply:
Testing on my live node now ...